### PR TITLE
Controllers creation

### PR DESCRIPTION
**Pull request by FacundoPucheta**
Created 2 new controllers: 
_serie_one_get()_   |   _serie_three_get()_

function required lines (at the top of the file) can be refactored as they will be implemented later

### CODE REVIEW

File: src/controllers/serie_one_get.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const probabilidades = require("../helpers/probabilidades.js");
```
Comment (manduinca) [inline src/controllers/serie_one_get.js]: Better: `probabilities`

File: src/controllers/serie_one_get.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const probabilidades = require("../helpers/probabilidades.js");
+
+
+const serie_one_get = async (location, periodo) => {  // periodo/lat/long obtained from input
```
Comment (manduinca) [inline src/controllers/serie_one_get.js]: Maybe: `const getExceedanceProbability` and `period`.

File: src/controllers/serie_three_get.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const e30_2003 = require("../helpers/e30_2003.js");
+const e30_2015 = require("../helpers/e30_2015.js");
+const e30_2015_esp = require("../helpers/e30_2015_esp.js");
+const ibc = require("../helpers/ibc.js");
+
+
+const serie_three_get = async (location, type, ground) => {
```
Comment (manduinca) [inline src/controllers/serie_three_get.js]: maybe: `getDesignSpectrum`

**Review by manduinca — DISMISSED** [review]
I approved, but you can improve it.

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by SebaSatelier — APPROVED** [review]

**Pull request merged**